adapters/postgres/queries.go

```diff
@@ -34,7 +33,7 @@ func (adapter *Postgres) GetScript(verb, folder, scriptName string) (script stri
 		return
 	}
 
-	script = filepath.Join(config.PrestConf.QueriesPath, folder, fmt.Sprint(scriptName, sufix))
```

Comment:
This change may be led to a problem.
The ```path.Join``` function will always use slashes as separators and I think the behavior of ```filepath.Join``` is more accurate here because that Join will use the OS specific separator.

Any thoughts @felipeweb @avelino ?